src/util/address.rs

```diff
@@ -405,6 +405,140 @@ impl Payload {
             } => script::Script::new_witness_program(version, prog)
         }
     }
+
+    /// Creates a pay to (compressed) public key hash payload from a public key
+    #[inline]
+    pub fn p2pkh(pk: &ecdsa::PublicKey) -> Payload {
+        Payload::PubkeyHash(PubkeyHash::hash(&pk.to_bytes()))
```

Comment:
Only now I understood that this should be `pk.pubkey_hash()` which is already a performance-optimized function doing no allocations

```suggestion
        Payload::PubkeyHash(pk.pubkey_hash())
```

Reply:
done